fedn/cli/run_cmd.py

```diff
@@ -137,7 +137,7 @@ def startup_cmd(ctx, path, keep_venv):
 
 @run_cmd.command("build")
 @click.option("-p", "--path", required=True, help="Path to package directory containing fedn.yaml")
-@click.option("-v", "--keep-venv", is_flag=True, default=True, required=False, help="flag if set to False doesn't remove venv")
+@click.option("-v", "--keep-venv", is_flag=True, required=False, help="flag if set to False doesn't remove venv")
```

Comment:
update "help": use flag to keep the python virtual environment (python_env in fedn.yaml). Deleting the venv is the default behavior